### PR TITLE
Import shared components

### PR DESCRIPTION
**Pull request by vincerubinetti**
This PR deletes most of `/components` and `/util`, and imports them from their new shared location at https://github.com/hetio/frontend-components

### CODE REVIEW

**Comment by dhimmel** [comment]
+82 −1,813 lines... nice. Will let @dongbohu do the actual code review, since it's out of my domain.

**Comment by vincerubinetti** [comment]
> +82 −1,813 lines... nice. Will let @dongbohu do the actual code review, since it's out of my domain.

I realized Dongbo is out today. I don't think it needs review though because it's literally just replacing `import` statements to point to the new package.

Working on this CI build failure now.

**Comment by vincerubinetti** [comment]
@dhimmel can you take a look at https://circleci.com/gh/greenelab/hetmech-frontend/442?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link and see if you have an idea of what's going wrong. It's having trouble installing the shared components package.

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,14 +3,13 @@ version: 2
...
-      - image: buildkite/puppeteer:v1.15.0
+      - image: node:8.15.1-stretch
     working_directory: ~/hetmech-frontend
     steps:
       - checkout
       - run:
           name: Frontend tests
           command: |
-            apt-get update && apt-get install procps --yes
             npm install
```
Comment (dhimmel) [inline .circleci/config.yml]: CI changes look good

**Pull request merged**